allennlp/commands/elmo.py

```diff
@@ -44,6 +44,8 @@
                            The batch size to use.
      --cuda-device CUDA_DEVICE
                            The cuda_device to run on.
+     --use-sentence-keys USE_SENTENCE_KEYS
+                           Whether to use line numbers or sentence keys as ids.
```

Comment:
I find this "sentence keys as ids" wording confusing, I might have said something like

"Normally a sentence's line number is used as the HDF5 key for its embedding. If this flag is specified, the sentence itself will be used as the key."